examples/customization/README.md

```diff
@@ -23,6 +23,7 @@ The table below summarizes all of the options. For some of them, there are examp
 | N/A | `http2` | Enables HTTP/2 in servers with SSL enabled. | `False` |
 | `nginx.org/redirect-to-https` | `redirect-to-https` | Sets the 301 redirect rule based on the value of the `http_x_forwarded_proto` header on the server block to force incoming traffic to be over HTTPS. Useful when terminating SSL in a load balancer in front of the Ingress controller — see [115](https://github.com/nginxinc/kubernetes-ingress/issues/115) | `False` | |
 | `ingress.kubernetes.io/ssl-redirect` | `ssl-redirect` | Sets an unconditional 301 redirect rule for all incoming HTTP traffic to force incoming traffic over HTTPS. | `True` | |
+| N/A | `error-log-level` | Sets the global error log level for NGINX. [error-log-level](http://nginx.org/en/docs/ngx_core_module.html#error_log).  | See the [template file](../../nginx-controller/nginx/nginx.conf.tmpl). | |
```

Comment:
Could you possible change the description to `Sets the global [error log level](http://nginx.org/en/docs/ngx_core_module.html#error_log) for NGINX.`  ?

Could you put `notice` in the default column instead of a link to the template?